File: app/src/main/java/com/sendbird/calls/quickstart/utils/ImageUtils.java
package com.sendbird.calls.quickstart.utils;

import android.content.Context;
import android.graphics.Bitmap;
import android.text.TextUtils;
import android.widget.ImageView;

import androidx.core.graphics.drawable.RoundedBitmapDrawable;
import androidx.core.graphics.drawable.RoundedBitmapDrawableFactory;

import com.bumptech.glide.Glide;
import com.bumptech.glide.request.RequestOptions;
import com.bumptech.glide.request.target.BitmapImageViewTarget;

public class ImageUtils {

    public static void displayCircularImageFromUrl(Context context, String imageUrl, ImageView imageView) {
        if (context == null || TextUtils.isEmpty(imageUrl) || imageView == null) {
            return;
        }

        Glide.with(context)
            .asBitmap()
            .apply(new RequestOptions().centerCrop().dontAnimate())
            .load(imageUrl)
            .into(new BitmapImageViewTarget(imageView) {
                @Override
                protected void setResource(Bitmap resource) {
                    RoundedBitmapDrawable circularBitmapDrawable = RoundedBitmapDrawableFactory.create(context.getResources(), resource);
                    circularBitmapDrawable.setCircular(true);
                    imageView.setImageDrawable(circularBitmapDrawable);
                }
            });
    }
}

File: directcall-kotlin/app/src/main/java/com/sendbird/calls/quickstart/utils/ImageUtils.kt
package com.sendbird.calls.quickstart.utils

import android.content.Context
import android.graphics.Bitmap
import android.widget.ImageView
import androidx.core.graphics.drawable.RoundedBitmapDrawableFactory
import com.bumptech.glide.Glide
import com.bumptech.glide.request.RequestOptions
import com.bumptech.glide.request.target.BitmapImageViewTarget

fun Context.displayCircularImageFromUrl(imageUrl: String?, imageView: ImageView) {
    Glide.with(this)
        .asBitmap()
        .apply(RequestOptions().centerCrop().dontAnimate())
        .load(imageUrl)
        .into(object : BitmapImageViewTarget(imageView) {
            override fun setResource(resource: Bitmap?) {
                val circularBitmapDrawable =
                    RoundedBitmapDrawableFactory.create(resources, resource)
                circularBitmapDrawable.isCircular = true
                imageView.setImageDrawable(circularBitmapDrawable)
            }
        })
}
